test/FakeMuxPort.cpp

```diff
@@ -36,7 +37,8 @@ FakeMuxPort::FakeMuxPort(
     std::string &portName,
     uint16_t serverId,
     boost::asio::io_service &ioService,
-    common::MuxPortConfig::PortCableType portCableType
+    common::MuxPortConfig::PortCableType portCableType,
+    bool useFakeLinkManagerStateMachine
```

Comment:
Why do we need a fake state machine? Is it relevant in this PR? When do we use it?

Reply:
when we want to test the state machine handler order, like the `MuxPortTest`